src/main/java/org/wildfly/security/auth/realm/ldap/OtpCredentialLoader.java

```diff
@@ -128,8 +129,8 @@ public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential
                 Password password = passwordFactory.generatePassword(new OneTimePasswordSpec(
                                 CodePointIterator.ofString((String) hashAttribute.get())
                                         .base64Decode(Alphabet.Base64Alphabet.STANDARD, false).drain(),
-                                CodePointIterator.ofString((String) seedAttribute.get())
-                                        .base64Decode(Alphabet.Base64Alphabet.STANDARD, false).drain(),
```

Comment:
My non-strong opinion is that we should not require padding on read, and always add it on write, which ensures correctness but tolerates user tinkering.  But, I'm not firm on that.